modules/openxAnalyticsAdapter.js

```diff
@@ -260,6 +292,7 @@ openxAdapter.originEnableAnalytics = openxAdapter.enableAnalytics;
 
 openxAdapter.enableAnalytics = function(config) {
   initOptions = config.options;
+  initOptions.testCode = checkTestCode();
```

Comment:
Just a nitpick: checkTestCode() should be broken out to something like `getValidTestCode` or default.  The action to check, doesn't tell us what actions are taking place.

Reply:
Got it! I'll update it.